src/components/common/data-not-found/data-not-found.directive.coffee

```diff
@@ -0,0 +1,63 @@
+module = angular.module('impac.components.common.data-not-found',[])
+
+module.directive('commonDataNotFound', ($templateCache, $log, $http, ImpacAssets, ImpacTheming, ImpacMainSvc) ->
+  return {
+    restrict: 'A',
+    scope: {
+      isKpi: '=?'
+      engine: '='
```

Comment:
I would rename that to `endpoint` instead to match `kpi.endpoint` (+ at some point `widget_category` will be renamed to `endpoint` as well...)